### DISCUSSION TITLE
Like buttons plugin

### DISCUSSION THREAD

**Discussion by giuseppelt** [incluud/accessible-astro-starter#63] [Ideas]
Hi, I created an [astro plugin](https://www.feelback.dev) which allows to add *Like* buttons to blog posts.
Are you open to integrate it in your template? I can create a PR to setup the integration. It will be just a few line of code. I will make the integration _optional_, meaning that if the `setting_key` is missing (the user doesn't want to use the plugin), nothing is affected and will change.

In general, the plugin allows to receive any kind of feedback, but I think for a blogsite _likes_ are more appropriate. Nonetheless, it's very easy and customizable.

Let me know.
Thanks.

**Comment by markteekman** [comment]
Hey @giuseppelt! Thanks for submitting your idea and thank you for your patience 🙂 The intention of the starter is to be as clean and minimal as possible, with a focus on accessibility, tools to enhance the developer experience and a minimal design for those who want something right out of the box. That said I'd like to keep the number of extensions that ship with the theme to a minimum so everyone can decide for themselves how to build their website on top of the basics.

The plugin looks really nice though, great work! 😃
